t/db/build_db.pl

```diff
@@ -115,27 +124,38 @@ sub addUsers {
 	return;
 }
 
-my $strp = DateTime::Format::Strptime->new(pattern => '%FT%T', on_error => 'croak');
-
 sub addSets {
 	# Add some problem sets
 	say 'adding problem sets' if $verbose;
 
-	my @hw_sets = loadCSV("$main::ww3_dir/t/db/sample_data/hw_sets.csv");
+	my @hw_sets = loadCSV(
+		"$main::ww3_dir/t/db/sample_data/hw_sets.csv",
+		{
+			boolean_fields       => ['set_visible'],
+			param_boolean_fields => [ 'enable_reduced_scoring', 'hide_hint' ]
+		}
+	);
 	for my $set (@hw_sets) {
 		my $course = $course_rs->find({ course_name => $set->{course_name} });
 		if (!defined($course)) {
 			croak 'The course ' . $set->{course_name} . ' does not exist';
 		}
-
+		$set->{set_params} = {} unless defined $set->{set_params};
```

Comment:
There is no need to set this.  The default will be filled in by the database if it is not defined.

Reply:
done.